js/reference.js

```diff
@@ -2832,7 +2832,27 @@ var ERMrest = (function(module) {
                 return null;
             }
 
-        }
+        },
+
+        /**
+         * @desc
+         * This function takes the current Tuple (this) and creates a shallow copy of it while de-referencing
+         * the _data attribute. This way _data can be modified in chaise without changing the originating Tuple
+         * @returns {ERMrest.Tuple} a shallow copy of _this_ tuple with it's _data de-referenced
+         */
+         deepCopyData: function() {
```

Comment:
I find the use of terms too confusing.

Change this function `deepCopyData` to `copy`

So the api will be 
```
var copiedTuple = thisTuple.copy();
```